apps/src/dance/ai/DanceAiModal.tsx

```diff
@@ -193,22 +185,18 @@ const DanceAiModal: React.FunctionComponent<DanceAiModalProps> = ({
   // (ie, the sum of ALL selected emoji's associations with a foreground/background palette combination).
   // Used to normalize and scale the data for easier differentiation between results by the user.
   const calculateMinMax = useCallback(
-    (useInputs: string[]) => {
+    (myInputs: string[]) => {
       // The minimum individual score is selected across all generated effects (bad and good).
       const minIndividualScore = getRangeArray(
         0,
         BAD_GENERATED_RESULTS_COUNT - 1
-      ).reduce((accumulator: number, currentValue: number) => {
-        const scores = getScores(useInputs, currentValue);
-        const min = Math.min(...scores);
-        return min < accumulator ? min : accumulator;
+      ).reduce((accumulator: number, step: number) => {
+        const scores = getScores(myInputs, step);
+        return Math.min(...scores, accumulator);
```

Comment:
This is just a logic simplification. It's returning the minimum of the previous value + all of the scores.

We could even remove the `scores` variable, but I felt it was more clear to keep.